source/plugins/quota.rst

```diff
@@ -0,0 +1,189 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with this work
+for additional information# regarding copyright ownership. The ASF licenses this
+file to you under the Apache License, Version 2.0 (the "License"); you may not
+use this file except in compliance with the License.  You may obtain a copy of
+the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by
+applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the specific language
+governing permissions and limitations under the License.
+
+
+Quota Plugin 
+=============
+
+Quota service, while allowing for scalability, will make sure that the cloud is
+not exploited by attacks, careless use and program errors. To address this
+problem, we propose to employ a quota-enforcement service that allows resource
```

Comment:
We don't propose anymore once this is out. "To address this problem, a quota-enforcement service can be employed. This service allows resource usage to be restricted to certain bounds as..."